README.md

```diff
@@ -214,6 +214,15 @@ general purpose passes have been upstreamed to mlir-hlo repository. We'll
 continue to work in a close cooperative relationship with mlir-hlo project in
 the longer term.
 
+### Roadmap with Torch-MLIR Project
+
+BladeDISC now maps PyTorch workload to MHLO based on [Torch-MLIR](https://github.com/llvm/torch-mlir/).
```

Comment:
I think it is not accurate to use the word bridge since it is bidirectional. We will not convert MHLO to Torch. I think the words map or convert are fitting.